<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResCap_Conversion</class>
 <widget class="QWidget" name="ResCap_Conversion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>實際電阻</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="0">
      <widget class="QLineEdit" name="capacitor_output_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_10">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>電容數值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_12">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>記號</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="capacitor_output_comboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>輸入</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="1" column="0">
      <widget class="QLineEdit" name="capacitor_Input_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>電容數值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>輸入</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLineEdit" name="Resistor_Input_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>電阻數值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>211</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>實際電阻</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <widget class="QLineEdit" name="Resistor_output_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_9">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>電阻數值</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_11">
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="text">
        <string>記號</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="Resistor_output_comboBox"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
